exporter/keys_test.go

```diff
@@ -52,6 +52,51 @@ func TestKeyValuesAndSizes(t *testing.T) {
 	}
 }
 
+func TestClusterKeyValuesAndSizes(t *testing.T) {
+	e, _ := NewRedisExporter(
+		os.Getenv("TEST_REDIS_CLUSTER_MASTER_URI"),
+		Options{Namespace: "test", CheckSingleKeys: dbNumStrFull + "=" + url.QueryEscape(keys[0]), IsCluster: true},
+	)
+
+	uri := os.Getenv("TEST_REDIS_CLUSTER_MASTER_URI")
+	if strings.Contains(uri, "://") {
+		url, _ := url.Parse(uri)
+		if url.Port() == "" {
+			uri = url.Host + ":6379"
+		} else {
+			uri = url.Host
+		}
+	} else {
+		if frags := strings.Split(uri, ":"); len(frags) != 2 {
+			uri = uri + ":6379"
+		}
+	}
+
```

Comment:
I don't think you need all of this, you can expect a correctly formed Uri for the tests.